WooCommerce/WooCommerceTests/ViewRelated/OrderPaymentDetailsViewModelTests.swift

```diff
@@ -0,0 +1,65 @@
+import XCTest
+@testable import WooCommerce
+@testable import Networking
+
+final class OrderPaymentDetailsViewModelTests: XCTestCase {
+    private var order: Order?
+    private var subject: OrderPaymentDetailsViewModel?
```

Comment:
non-blocking suggestion: I wonder if it's worth unwrapping these two optional variables here instead of various places in the functions below? They are set for sure in every `setUp` call

Reply:
TIL! I've always assumed that in order to nullify then in tearDown, I'd need them to be declared as optional, and I've never given it a second thought. Thanks!